github/Requester.py

```diff
@@ -372,9 +383,6 @@ def __requestRaw(self, cnx, verb, url, requestHeaders, input):
         original_cnx = cnx
         if cnx is None:
             cnx = self.__createConnection()
-        else:
```

Comment:
We don't have to check for the magic string value anymore, because we already get the proper custom connection to `status.github.com` passed if we are connecting to status API